resources/lang/en/settings.php

```diff
@@ -123,6 +123,7 @@
     'audit_table_date' => 'Activity Date',
     'audit_date_from' => 'Date Range From',
     'audit_date_to' => 'Date Range To',
+    'ip_address' => 'IP',
```

Comment:
We already have the `audit_table_ip` string above. Could we just use that translation instead for now? I think showing "IP Address" as the label is better suited while aligning with the table, plus it'll reduce required translations.

Reply:
Thanks for reminding me. I did not notice there is a prefix.